packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -97,6 +100,7 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
 
   @override
   void dispose() {
+    _shownFirstScript = false;
```

Comment:
no need to clear in dispose as disposed objects are invalid.

Reply:
done